sql/src/main/java/io/crate/operation/projectors/ShardingUpsertExecutor.java

```diff
@@ -326,9 +329,8 @@ private void processShardResponse(ShardResponse shardResponse) {
         return listener;
     }
 
-    private Map<ShardLocation, TReq> getFromPendingToRequestMap(
-        Map<String, List<PendingRequest<TItem>>> requestsByIndexForCurrentBulk) {
-        Map<ShardLocation, TReq> requests = new HashMap<>();
+    private void fromPendingToRequestMap(
```

Comment:
is this better? the method name and parameters are not expressive anymore ( method saying from something to something else but doesn't return anything)